xml/docu_styleguide.structure.xml

```diff
@@ -2667,10 +2667,79 @@ xml:id="tab-install-source"</screen>
    <tag class="attribute">xml:id</tag>
    attribute.
   </para>
-  <para>
-   Value and description pairs are better handled by means of a descriptive
-   list.
-  </para>
+
+  <para>Typical use cases of tables include:</para>
+  
+  <variablelist>
+   <varlistentry><term>Lookup tables for specific information</term>
+    <listitem>
+     <para>Whenever users need to check for data that applies to them, 
+      create lookup tables. For example, use a table to sum up system requirements.
+      </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry><term>Matrix tables</term>
+    <listitem>
+     <para>Whenever users need to quickly check whether a specific combination
+      of options works or not. For example, use a matrix table to visualize supported 
+      combinations or update options.</para>
+    </listitem>
+   </varlistentry>
+  </variablelist>
+
+  <para>As there are use cases for tables, there are cases when <emphasis>not</emphasis> 
+   to use tables:</para>
+  
+  <itemizedlist>
+   <listitem>
+    <para>Value and description pairs are better handled by means of a descriptive list.</para>
+   </listitem>
+   <listitem>
+    <para>Wordy explanations or descriptions should not be used in tables.</para>
+   </listitem>
+   <listitem>
+    <para>Complex layout constructs (screens, code) should not be used in tables.</para>
+   </listitem>
+  </itemizedlist>
+
+  <para>Some style tips for tables:</para>
+  
+  <variablelist>
+   <varlistentry><term>Structure the table to have more rows than columns</term>
+    <listitem>
+     <para>Readers prefer to skim for information that is aligned vertically. When designing 
```

Comment:
Maybe as an (optional) addition, you could mention:

```suggestion
     <para>Readers prefer to skim for information that is aligned vertically.
       It is easier to read a table without scrolling horizontally. When designing 
```

Or add it at the last sentence. Or don't change anything. Up to you.

Reply:
I prefer to leave stuff as it is :) But thanks for the suggestion anyway!